Provisioning/Vault/audit.tf

```diff
@@ -1,7 +1,17 @@
-resource "vault_audit" "file" {
+resource "vault_audit" "stdout" {
   type = "file"
 
   options = {
     file_path = "stdout"
   }
-}
\ No newline at end of file
+}
+
+resource "vault_audit" "otel" {
```

Comment:
This tcp audit device is the "primary" audit device in our example here. Obviously, one could also decided to declare the "stdout" logging path as the primary audit log path and only use the tcp device here for backup purposes. It depends a bit on your requirements.